doc/modules/kernel_approximation.rst

```diff
@@ -154,6 +154,24 @@ above for the :class:`RBFSampler`. The only difference is in the free
 parameter, that is called :math:`c`.
 For a motivation for this mapping and the mathematical details see [LS2010]_.
 
+TensorSketch Method for Polynomial Kernel Approximation
+--------------------------------------------------------
+
+The TensorSketch [PP2013]_ method, as implemented in :class:`PolynomialSampler`, is a
+scalable, data-independent method for polynomial kernel approximation. This method
+obtains a Count Sketch [CCF2002]_ of the outer product of two vectors (or a vector with
+itself), which an be used as an approximation of the polynomial kernel feature space.
+In particular, instead of explicitly computing the outer product, TensorSketch
+computes the Count Sketch of the vectors and then uses polynomial multiplication
+via the Fast Fourier Transform to compute the Count Sketch of their outer product.
+
+Conveniently, the training phase of TensorSketch simply consists of initializing
+some random variables, thus being a data-independent method. In addition, this
+method can transform samples in :math:`\mathcal{O}(n(d + k \text{log} k))`, where
+:math:`n` is the number of samples, :math:`d` the dimension of input samples
+and :math:`k` the desired output dimension, determined by (``n_components``). 
+
+.. _tensor_sketch_kernel_approx:
```

Comment:
this narrative doc here should point the example that demos the usage with the figure produced by it.
can you take care of it?

also I would really suggest to simplify the example so it demos the usage and we keep the full benchmark for the benchmarks folder here: https://github.com/scikit-learn/scikit-learn/tree/master/benchmarks

Reply:
Yes, sure.

I will move the current example to the benchmarks folder. I am working on a simpler example with a real-world dataset, as you suggested.